test/stat_tracker_test.rb

```diff
@@ -2,29 +2,27 @@
 require './lib/stat_tracker'
 
 class StatTrackerTest < Minitest::Test
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+  def setup ## instantiate using the from_csv
+    @game_path = './data/games.csv'
```

Comment:
We need to remember to go back later and take out commented out text, just when it becomes unnecessary.